.github/workflows/tools.yml

```diff
@@ -142,12 +142,10 @@ jobs:
             subsystem: deps
             label: dependencies
             run: |
-              NEW_VERSION=$(gh api repos/nghttp2/nghttp2/releases/latest -q '.tag_name|ltrimstr("v")')
-              CURRENT_VERSION=$(grep "#define NGHTTP2_VERSION" ./deps/nghttp2/lib/includes/nghttp2/nghttp2ver.h | sed -n "s/^.*VERSION \(.*\)/\1/p")
-              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
-                echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
-                ./tools/update-nghttp2.sh "$NEW_VERSION"
-              fi
+              ./tools/dep_updaters/nghttp2.sh > temp-output
```

Comment:
Should be `tools/dep_updaters/update-nghttp2.sh`.
https://github.com/nodejs/node/issues/47326